package.json

```diff
@@ -13,7 +13,9 @@
   },
   "author": "Stack Education",
   "license": "ISC",
-  "dependencies": {},
+  "dependencies": {
```

Comment:
Looks like you ran `npm install i`. Remember it is either `npm install` or `npm i` not both.